benchmark.py

```diff
@@ -120,10 +120,9 @@ def bson_numpy_func(use_large):
         collection_names[use_large],
         codec_options=CodecOptions(document_class=RawBSONDocument))
 
-    cursor = raw_coll.find()
+    cursor = raw_coll.find_raw_batches()
     dtype = dtypes[use_large]
-    bsonnumpy.sequence_to_ndarray(
-        (doc.raw for doc in cursor), dtype, raw_coll.count())
+    bsonnumpy.sequence_to_ndarray(cursor, dtype, raw_coll.count_documents({}))
```

Comment:
This change defeats the purpose of the this benchmark which is to test the performance of `coll.find()` with RawBSONDocuments. Is this the part that segfaults?

Reply:
I was unable to find another way to make it work, so I would suggest removing this benchmark.